src/vs/workbench/contrib/testing/browser/testResultsView/testResultsOutput.ts

```diff
@@ -232,7 +231,7 @@ export class MarkdownTestMessagePeek extends Disposable implements IPeekOutputRe
 			return undefined;
 		}
 
-		this.element.style.width = `${dimension.width}px`;
+		this.element.style.width = `${dimension.width - 32}px`;
```

Comment:
How was the `32` calculated?

Reply:
This is the horizontal margin of the `.test-output-peek-message-container .preview-text`